src/Propulsion.SqlStreamStore/SqlStreamStoreSource.fs

```diff
@@ -0,0 +1,71 @@
+namespace Propulsion.SqlStreamStore
+
+open System
+open SqlStreamStore
+
+type SqlStreamStoreSource =
+
+    static member internal RunInternal
+            (logger : Serilog.ILogger,
+             store: IStreamStore,
+             checkpointer: ICheckpointer,
+             sink : Propulsion.ProjectorPipeline<_>,
+             consumerGroup: string,
+             ?statsInterval: TimeSpan,
+             ?readerMaxBatchSize: int,
+             ?readerSleepInterval: TimeSpan,
+             ?readerStopAfterInterval: TimeSpan) : Async<unit> =
+
+        async {
+            let streamId = "$all"
+
+            let logger =
+                let instanceId = System.Guid.NewGuid()
+                logger
+                    .ForContext("instanceId", string instanceId)
+                    .ForContext("consumerGroup", consumerGroup)
+
+            let ingester : Propulsion.Ingestion.Ingester<_,_> =
+                sink.StartIngester(logger, 0)
+
+            let reader =
+                StreamReader(logger,
+                             store,
+                             checkpointer,
+                             ingester.Submit,
+                             streamId,
+                             consumerGroup,
+                             ?maxBatchSize = readerMaxBatchSize,
+                             ?sleepInterval = readerSleepInterval,
+                             ?statsInterval = statsInterval,
+                             ?stopAfterInterval = readerStopAfterInterval)
+
+            try
+                let! position =
+                    checkpointer.GetPosition { Stream = streamId; ConsumerGroup = consumerGroup }
+
+                do! reader.Start(position)
+            finally
+                ingester.Stop()
+        }
+
+    /// Run SqlStreamStore.
+    static member Run
+        (logger : Serilog.ILogger,
+         store: IStreamStore,
```

Comment:
nits (most of these are in F# formatting guidelines, many in `.editorconfig`): 
- always have a CR at eof
- all decls should have spaces around `:`, i.e., `store : IStreamStore`
- commas in arg lists, tuple creations etc should have spaces before
- `spaces after `;`, e.g. `[<NoComparison; NoEquality>]`
- no space before method args (vs function args which always have space before), e.g. `logger.Information("message")`
- naming of args in arg lists should not have surrounding spaces, i.e. `?readerMaxBatchSize=readerMaxBatchSize`
- use `[]` in preference to `array`

Reply:
I made some changes here, but beyond that feel free to run whatever linter/formatter you use ;)

Reply:
Thanks, as mention it all looks pretty consistent now (as it happens I've realised only today that this repo's spacing around colons is not in accordance with https://docs.microsoft.com/en-us/dotnet/fsharp/style-guide/formatting - I for one welcome our incoming final linter overlord!)